experiments/jz/joint_training_toy/joint_training_toy1/02_load_dataset.slurm

```diff
@@ -57,6 +57,7 @@ cd $WORK/repos/sync/metadata/
 
 # We check that the dataset can indeed be loaded
 python experiments/jz/utils/loading_script_utils/load_dataset.py \
-    dataset_name="${REPO_DIR}" \
-    train_file="XXX" \ # (change me and remove the comment! e.g "nq-train-*.jsonl.gz" or remove arg)
-    validation_file="XXX" # (change me and remove the comment! e.g. "nq-dev-*.jsonl.gz" or remove arg)
+    dataset_name="${REPO_DIR}" 
+    # \
+    # train_file="XXX" \ # (change me and remove the comment! e.g "nq-train-*.jsonl.gz" or remove arg)
+    # validation_file="XXX" # (change me and remove the comment! e.g. "nq-dev-*.jsonl.gz" or remove arg)
```

Comment:
It seems to me that we still need to specify the files we want to include here (so that the automatic data format detection feature of the datasets library works)

For example by putting:
```
train_file="c4-train*"
```

Reply:
I tried it and it seems to work the same with or without that argument.